driver-core/src/test/functional/com/mongodb/operation/AggregateOperationSpecification.groovy

```diff
@@ -191,7 +193,7 @@ class AggregateOperationSpecification extends OperationFunctionalSpecification {
         async << [true, false]
     }
 
-    @IgnoreIf({ !(serverVersionAtLeast(3, 6) && isDiscoverableReplicaSet()) })
+    @IgnoreIf({ !(serverVersionAtLeast(3, 6) && !isStandalone()) })
```

Comment:
Should put this in a separate commit.   But the change is because we can test change streams against sharded clusters now and we missed this spot.

Reply:
No its fine here.